docs/dev/opensearch-same-table-join-query.md

```diff
@@ -0,0 +1,1252 @@
+# Overview
+
+The OpenSearch SQL plugin provides a thin-client language-processing engine from SQL to an OpenSeach instance.  The SQL
+language is a relational language that provides the JOIN feature to relate tables together by field or expression.  OpenSearch
+offers a similar relation mapping within an index, that allows users to relate same-index documents using a parent-child
```

Comment:
Maybe we can link to the documentatin page for users looking for a more in-depth description of the mapping type.

Reply:
Updated.  Thanks